tests/core/test_export_helpers.py

```diff
@@ -164,11 +164,11 @@ def test_xlsx_export(tmpdir):
         },
     ],
 )
+@pytest.mark.unit
```

Comment:
@SteveDMurphy this test was actually getting missed, and was failing, I'm trying to figure out what to fix here but I remember you made some changes to the DataSubjectRights

Reply:
Looking into this more, great catch! 😅 
Will open up a separate bug issue for this to get it sorted as both the `ALL` and `NONE` options are not working as is... feel free to remove the test for now (or leave unmarked apparently)

Reply:
this has been fixed and will be merged into this PR before merging